docker/datahub-mce-consumer/env/docker.env

```diff
@@ -1,8 +1,20 @@
 MCE_CONSUMER_ENABLED=true
+EBEAN_DATASOURCE_USERNAME=datahub
```

Comment:
One obvious downside of this is that the standalone consumers have more entanglement, less loose coupling with the state stores.

The complexity of maintaining this is something to think about - are the gains worth it?

Reply:
I would like to merge the 3 subcharts in helm one day to reduce configuration complexity. Really mce/mae/gms are tightly coupled and the same configuration should always be applied to all three. Fortunately even though the helm subcharts are separate the configuration is global so no additional helm chart values are required!

I believe this is the key to scaling ingestion horizontally and separating impact of ingestion on the GMS graphql read path for the UI.